test/db_task_history_test.py

```diff
@@ -82,3 +82,24 @@ def run_task(self, task):
         self.history.task_scheduled(task.task_id)
         self.history.task_started(task.task_id, 'hostname')
         self.history.task_finished(task.task_id, successful=True)
+
+
+class MySQLDbTaskHistoryTest(unittest.TestCase):
+
+    @with_config(dict(task_history=dict(db_connection='mysql+pymysql://travis@localhost/luigi_test')))
```

Comment:
We are already using [mysql-connector](https://github.com/spotify/luigi/blob/master/luigi/contrib/mysqldb.py#L25), maybe we could stick with it? (You would need to modify tox to add `--allow-external mysql-connector-python` to [install_command](https://tox.readthedocs.org/en/latest/config.html?highlight=install_command#confval-install_command=ARGV))